packages/blockchain-wallet-v4/src/network/api/simpleBuy/index.ts

```diff
@@ -420,6 +425,19 @@ export default ({
       }
     })
 
+  const getSBLimits = (
+    currency: FiatType,
+    product: NabuCustodialProductType,
+    networkFee: CoinType,
+    side: SBOrderActionType
+  ): SwapUserLimitsType =>
+    authorizedGet({
+      url: nabuUrl,
+      endPoint: `/trades/limits?currency=${currency}&product=${product}&networkFee=${networkFee}&side=${side}`,
+      contentType: 'application/json',
+      ignoreQueryParams: true
+    })
```

Comment:
My thought is to no matter as we call same and point business logic should remain in product so all related simpleBuy should be here etc...

Reply:
my first thought was that these two requests could just be merged, but I see your logic. If we merged them into one call should the api endpoint live in the simple buy apis dir or the swap apis dir, or do we create a new "generic" dir for this one endpoint? In the end I think you're right that we just duplicate this in order to keep the business logic on the client and not over abstract the code. 👍🏻

Reply:
One improvement you could make to this is to make the arguments a single object called `data` and can be passed directly to the `authorizedGet` object and it'll turn them in to proper query params so your url can be way shorter.

```
const getSBLimits = (data: SwapUserLimitsType) =>
    authorizedGet({
      url: nabuUrl,
      endPoint: `/trades/limits`,
      contentType: 'application/json',
      data,
      ignoreQueryParams: true,
    })
```

Reply:
I like this one, yeah can apply it